beams/behavior_tree/ActionNode.py

```diff
@@ -84,17 +84,18 @@ def update(self) -> py_trees.common.Status:
 
         return new_status
 
-    def terminate(self, new_status: py_trees.common.Status) -> None:
-        """Nothing to clean up in this example."""
-        print(f"TERMINATE CALLED ON {self.name}, pid: {os.getpid()}")
-        if self.work_proc.is_alive():
-            print(f"The process is still alive on {os.getpid()}")
-            self.work_proc.terminate()
-            self.logger.debug(
-                py_trees.console.red
-                + "%s.terminate()[%s->%s]"
-                % (self.__class__.__name__, self.status, new_status)
-            )
+    # TODO: serious introspection about that we want to do here. 
+    # def terminate(self, new_status: py_trees.common.Status) -> None:
+    #     """Nothing to clean up in this example."""
+    #     print(f"TERMINATE CALLED ON {self.name}, pid: {os.getpid()}")
+    #     if self.work_proc.is_alive():
+    #         print(f"The process is still alive on {os.getpid()}")
+    #         self.work_proc.terminate()
+    #         self.logger.debug(
+    #             py_trees.console.red
+    #             + "%s.terminate()[%s->%s]"
+    #             % (self.__class__.__name__, self.status, new_status)
+    #         )
```

Comment:
Given your above comment, maybe we can put in a basic log message instead of wiping the function entirely? I'm not sure precisely which info is useful here.
```suggestion
    def terminate(self, new_status: py_trees.common.Status) -> None:
        """Nothing to clean up.""
        logger.debug("%s terminated", self)
```